Library/DasharoSystemFeaturesUiLib/DasharoSystemFeatures.c

```diff
@@ -24,13 +24,21 @@ STATIC CHAR16 mUsbMassStorageEfiVar[] = L"UsbMassStorage";
 STATIC CHAR16 mPs2ControllerEfiVar[] = L"Ps2Controller";
 STATIC CHAR16 mWatchdogEfiVar[] = L"WatchdogConfig";
 STATIC CHAR16 mWatchdogStateEfiVar[] = L"WatchdogAvailable";
-
-STATIC BOOLEAN mUsbStackDefault = TRUE;
-STATIC BOOLEAN mUsbMassStorageDefault = TRUE;
-STATIC BOOLEAN mLockBiosDefault = TRUE;
-STATIC BOOLEAN mSmmBwpDefault = FALSE;
-STATIC UINT8   mMeModeDefault   = ME_MODE_ENABLE;
-STATIC BOOLEAN mPs2ControllerDefault = TRUE;
+STATIC CHAR16 mFanCurveEfiVar[] = L"FanCurve";
+STATIC CHAR16 mFanCurveOptionEfiVar[] = L"FanCurveOption";

```

Comment:
Why 2 variables? I thought we will just keep the profile number

Reply:
This would make eventually implementing custom fan curves easier, but i guess i can remove it now to simplify

Reply:
removed